src/util/linkRelationships/index.js

```diff
@@ -24,8 +25,8 @@ async function link (id, type, included) {
  */
 async function linkArray (data, included, key) {
   data[key] = []
-  for (let { id, type } of await data.relationships[key].data) {
-    data[key].push(await link(id, type, included))
+  for (let { id, type, meta } of await data.relationships[key].data) {
+    data[key].push(await link(id, type, meta, included))
```

Comment:
At this point its simpler to just dump the `resource` object into the `link` function instead of destructuring more properties

```js
for (let resource of await data.relationships[key].data) {
	data[key].push(await link(resource, included))
}
```